### ISSUE TITLE
nix-indent-line gets confused by a comment containing "in"

### ISSUE THREAD

**Issue by grahamc** [NixOS/nix-mode#62]
```nix
let
  foo = bar:
    # comment containing in 
  # dedented 
  bar;
in null
```

should be 

```nix
let
  foo = bar:
    # comment containing in 
    # dedented 
    bar;
in null
```

**Comment by grahamc** [comment]
Additionally:

```nix
{
  # with
    foo = "bar";
}
```

**Comment by grahamc** [comment]
Fixed as of #68!
